catalogue/webapp/pages/works.tsx

```diff
@@ -155,39 +155,37 @@ const Works: NextPage<Props> = ({ works, worksRouteProps }) => {
                 <div className="grid">
                   <div className={grid({ s: 12, m: 12, l: 12, xl: 12 })}>
                     <PaginationWrapper>
-                      <Fragment>
-                        <Paginator
-                          query={query}
-                          showPortal={true}
-                          currentPage={page}
-                          totalPages={works.totalPages}
-                          totalResults={works.totalResults}
-                          link={toLink(
+                      <Paginator
```

Comment:
This looks messier than it actually is; I just removed the wrapper `<Fragment>` tags; they're unnecessary.